Armadillo/src/main/java/com/scribd/armadillo/playback/PlayerEventListener.kt

```diff
@@ -33,8 +34,8 @@ internal class PlayerEventListener : Player.EventListener {
     @Inject
     internal lateinit var stateModifier: StateStore.Modifier
 
-    override fun onPlayerError(error: ExoPlaybackException) {
-        val exception = error.toArmadilloException(error)
+    override fun onPlayerError(error: PlaybackException) {
+        val exception = (error as ExoPlaybackException).toArmadilloException()
```

Comment:
It should be okay to perform this unsafe cast here as we are using exoplayer as our engine.

See this excerpt.

```
Player implementations like ExoPlayer may use PlaybackException subclasses (like ExoPlaybackException), so users can downcast the PlaybackException instance to obtain implementation-specific fields (like ExoPlaybackException.rendererIndex).
```
https://github.com/google/ExoPlayer/blob/release-v2/RELEASENOTES.md#2150-2021-08-10